### PR TITLE
Recipe for JSON::Any

### PR DESCRIPTION
**Pull request by pvanheus**
This is part of the dependencies for the [SNVPhyl](https://snvphyl.readthedocs.io/en/latest/) workflow.

### CODE REVIEW

**Comment by jakirkham** [comment]
Not sure if these overlap with any of the one's you are interested in, @epruesse, but figured you might like to know about this PR.

**Comment by jakirkham** [comment]
@bgruening, you seem to know quite a bit about `perl`. Would be great to hear your thoughts on these?

File: recipes/perl-string-util/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+requirements:
+  build:
+    - perl
+    - make
+    - perl-number-misc
+    - perl-test-toolbox
+  run:
+    - perl
+    - perl-number-misc
+    - perl-test-toolbox
```
Comment (bgruening) [inline recipes/perl-string-util/meta.yaml]: Is this really needed as runtime dependency?

File: recipes/perl-try-tiny/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - perl
+
+test:
+  commands:
+    - perl -V
+    - perl -e 'use {{ name|replace("-","::") }}'
+
+about:
+  home: https://metacpan.org/release/Try-Tiny
+  license: Artistic-2.0
```
Comment (bgruening) [inline recipes/perl-try-tiny/meta.yaml]: according to the website http://search.cpan.org/~ether/Try-Tiny-0.28/lib/Try/Tiny.pm this is MIT licensed?

**Comment by bgruening** [comment]
@jakirkham looks very good to me. Did you mean anything else beside a review?

File: recipes/perl-try-tiny/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/perl-try-tiny/meta.yaml]: Pinging this issue!

**Pull request closed without merging**